omod/src/main/webapp/fragments/field/concept.gsp

```diff
@@ -0,0 +1,58 @@
+<%
+    config.require("formFieldName")
+    config.require("conceptId")
+%>
+
+<script type="text/javascript">
+    jq(function() {
+        var xhr=null;//used to track active ajax requests
+        
+        jq('#${ config.id }-field').autocomplete({
+            source: function(request, response) {
+                var ajaxUrl = '${ ui.actionLink("registrationapp", "concept", "getConcepts")}';
```

Comment:
Why do you have to make a request to the server, can't you store the answers in a JS variable?